MantidQt/CustomInterfaces/src/Muon/MuonAnalysisResultTableTab.cpp

```diff
@@ -497,40 +499,33 @@ void MuonAnalysisResultTableTab::populateLogsAndValues(
     // Get log information
     std::string wsName = fittedWsList[i].toStdString();
     auto ws = retrieveWSChecked<ExperimentInfo>(wsName + WORKSPACE_POSTFIX);
-
-    Mantid::Kernel::DateAndTime start = ws->run().startTime();
-    Mantid::Kernel::DateAndTime end = ws->run().endTime();
-
     const std::vector<Property *> &logData = ws->run().getLogData();
+    const TimeSeriesProperty<bool> *runningLog = nullptr;
+    Property *runLog = nullptr;
+    const bool foundRunning = ws->run().hasProperty(running);
+    if (foundRunning) {
+      runLog = ws->run().getLogData(running);
+      runningLog = dynamic_cast<TimeSeriesProperty<bool> *>(runLog);
 
+    } else {
+      Mantid::Kernel::Logger g_log("MuonAnalysisResultTableTab");
+      g_log.warning(
+          "No running log found. Filtering will not be applied to the data.\n");
+    }
     for (const auto prop : logData) {
```

Comment:
I know this isn't something you have changed but I would suggest adding a `&` qualifier to the `for` loop. It's only copying a pointer but I think it's good practice to avoid copies where possible:

```
for (const auto &prop : logData)
```